tests/test_layer_parameters.py

```diff
@@ -0,0 +1,191 @@
+import pytest
+from pyproj import CRS
+from city_metrix.layers import (
+    Layer,
+    Albedo,
+    AlosDSM,
+    AverageNetBuildingHeight,
+    BuiltUpHeight,
+    EsaWorldCover, EsaWorldCoverClass,
+    LandSurfaceTemperature,
+    NasaDEM,
+    NaturalAreas,
+    NdviSentinel2,
+    TreeCanopyHeight,
+    TreeCover,
+    UrbanLandUse,
+    WorldPop, OpenStreetMap, HighLandSurfaceTemperature
+)
+from tests.resources.bbox_constants import BBOX_BRA_LAURO_DE_FREITAS_1
+from tests.tools.general_tools import get_class_from_instance, get_class_default_spatial_resolution
+
+"""
+Evaluation of spatial_resolution property 
+To add a test for a scalable layer that has the spatial_resolution property:
+1. Add the class name to the city_metrix.layers import statement above
+2. Copy an existing test_*_spatial_resolution() test
+   a. rename for the new layer
+   b. specify a minimal class instance for the layer, not specifying the spatial_resolution attribute.
+"""
+
+COUNTRY_CODE_FOR_BBOX = 'BRA'
+BBOX = BBOX_BRA_LAURO_DE_FREITAS_1
+RESOLUTION_TOLERANCE = 1
+
+def test_albedo_spatial_resolution():
+    class_instance = Albedo()
+    doubled_default_resolution, actual_estimated_resolution =  evaluate_doubled_resolution_property(class_instance)
+    assert pytest.approx(doubled_default_resolution, rel=RESOLUTION_TOLERANCE) == actual_estimated_resolution
```

Comment:
It might also be useful to test the accuracy of the data after resampling to a tolerance. That might catch cases where certain layers should use specific resampling methods to ensure accuracy (I'm assuming it uses nearest resampling by default).

Reply:
Good point. I'll look into testing it.

Reply:
It's going to take a while to develop a robust layer comparison tool, such as using RMSE. I'll open a separate ticket for that. 

I have visually inspected all of the down-sampled rasters and believe the results are good.

Reply:
I implemented test_layer_parameter.evaluate_raster_value function and believe this was very worthwhile.

Reply:
I'll resolve this here but see below